copter/source/docs/traditional-helicopter-autotune.rst

```diff
@@ -0,0 +1,199 @@
+| +-------+-----------------+ | |         | & Rate P             | |
+| | 3     | Max Gain        | | +---------+----------------------+ |
+| +-------+-----------------+ | | 4       | Angle P              | |
+|                             | +---------+----------------------+ |
+|                             | | 5       | Feedforward & Angle P| |
+|                             | +---------+----------------------+ |
+|                             | | 6       | Rate D*, Rate P,     | |
+|                             | |         | & Angle P            | |
+|                             | +---------+----------------------+ |
+|                             | | 7       | Feedforward, Rate D*,| |
+|                             | |         | Rate P, & Angle P    | |
+|                             | +---------+----------------------+ |
+|                             | | 8       | Max Gain             | |
+|                             | +---------+----------------------+ |
++-----------------------------+------------------------------------+
+
+Max gain determination is done before tuning Rate D and Rate P gains.
+
+:ref:`Axes Bitmask<AUTOTUNE_AXES__AC_AutoTune_Heli>`
+-----------------------------------------------------------
+
+Specifies one or more axes to be tuned.
+
++----------------------+---------------------------------+
+| Bits                 | Values                          |
++======================+=================================+
+| +-------+----------+ | +---------+-------------------+ |
+| | Bits  | Axis     | | | Values  | Axes tested       | |
+| +=======+==========+ | +=========+===================+ |
+| | 0     | Roll     | | | 1       | Roll              | |
+| +-------+----------+ | +---------+-------------------+ |
+| | 1     | Pitch    | | | 2       | Pitch             | |
+| +-------+----------+ | +---------+-------------------+ |
+| | 2     | Yaw      | | | 3       | Roll & Pitch      | |
+| +-------+----------+ | +---------+-------------------+ |
+|                      | | 4       | Yaw               | |
+|                      | +---------+-------------------+ |
+|                      | | 5       | Roll & Yaw        | |
+|                      | +---------+-------------------+ |
+|                      | | 6       | Pitch & Yaw       | |
+|                      | +---------+-------------------+ |
+|                      | | 7       | Roll, Pitch & Yaw | |
+|                      | +---------+-------------------+ |
++----------------------+---------------------------------+
+
+:ref:`Maximum Response Gain<AUTOTUNE_GN_MAX>`
+---------------------------------------------
+
+Specifies the maximum response gain to be used to tune the Rate D, Rate P and Angle P gains.  The response gain is the output of the response divided by the input.  It is recommended that 1.8 to 2.0 be used for pitch and roll axes as these axes are typically more lightly damped.  It is recommended that 1.0 to 1.4 be used for the yaw axis.
```

Comment:
how does this relate to the maximum gain in the table above?

Reply:
The comment in the table above for the max gain test determines the max allowable rate P and rate D gains.   This parameter is used to tune the rate P and rate D gains and the tuning stops when this response gain is exceeded.

Reply:
see my comment above...I think that its automatically added as a preceding  tune for the D/P tune correct? how is it autotuned by itself? still confused

Reply:
It is automatically added before the Rate P and Rate D tuning.  What Max gains does is perform a test with no rate P or rate D gain.  the max allowable Rate P and Rate D gains are mathematically determined from the results of this test.  Then it uses 5% increments of these max gains to increment Rate P and Rate D gains for tuning these parameters.

Reply:
@Hwurzburg I made a change that I think works.